build.sbt

```diff
@@ -33,6 +33,14 @@ lazy val root = (project in file("."))
     Test / sources     := Seq.empty
   )
 
+lazy val clientCI = (project in file("client/target/client_ci"))
+  .settings(commonSettings, clientSettings, publishCISettings)
+  .settings(
+    Compile / scalaSource := baseDirectory.value / "client" / "src" / "main" / "scala",
+    releaseVersionFile := baseDirectory.value / ".." / "version.sbt",
+  )
+  .enablePlugins(BuildInfoPlugin)
+
```

Comment:
now it's proven, would be nice to remove the superfluous project

Reply:
We tried removing the project but actually there are some underlying processes/steps that aren't quite compatible (yet, or without further work on our part).

Would the intention be to only support CI releases of the project, or would you always want to allow both mechanisms for release (via CI and manually)?

Reply:
Actually, not to worry - we have a solution that integrates with the existing project now :)